src/tools/ceph-dencoder/osd_types.h

```diff
@@ -170,6 +170,7 @@ TYPE(mon_feature_t)
 TYPE_FEATUREFUL(DataStats)
 TYPE_FEATUREFUL(ProgressEvent)
 TYPE(FeatureMap)
+TYPE(PoolAvailability)
```

Comment:
Please let me know if it should be `TYPE_FEATUREFUL` instead. Wasn't sure what is the difference between `TYPE` and `TYPE_FEATUREFUL`, @NitzanMordhai @kamoltat could you folks please help me understand / point me to docs?

Reply:
@shraddhaag since you are not depending on FEATUREFUL bits, you don't need to set it as a TYPE_FEATUREFUL